airbyte-webapp/src/components/connection/CatalogTree/next/StreamFieldsTable/StreamFieldsTable.tsx

```diff
@@ -0,0 +1,206 @@
+import { faArrowRight } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { createColumnHelper } from "@tanstack/react-table";
+import React, { useMemo } from "react";
+import { FormattedMessage, useIntl } from "react-intl";
+
+import { SyncSchemaField, SyncSchemaFieldObject } from "core/domain/catalog";
+import { AirbyteStreamConfiguration } from "core/request/AirbyteClient";
+import { useConnectionFormService } from "hooks/services/ConnectionForm/ConnectionFormService";
+import { useDestinationDefinition } from "services/connector/DestinationDefinitionService";
+import { useSourceDefinition } from "services/connector/SourceDefinitionService";
+import { equal } from "utils/objects";
+import { getDataType } from "utils/useTranslateDataType";
+
+import { CheckBox } from "../../../../ui/CheckBox";
+import { RadioButton } from "../../../../ui/RadioButton";
```

Comment:
Could you fix imports?

import { CheckBox } from "components/ui/CheckBox";
import { RadioButton} from "components/ui/RadioButton";

Reply:
Fixed